pkg/interceptors/cel/cel_test.go

```diff
@@ -735,21 +735,21 @@ func TestExpressionEvaluation_Error(t *testing.T) {
 			expr: "body.value.parseJSON().test == 'test'",
 			want: "invalid character 'e' in literal",
 		},
-		{
-			name: "base64 decoding non-string",
-			expr: "body.pull_request.decodeb64()",
-			want: "unexpected type 'map' passed to decodeB64",
-		},
-		{
-			name: "parseJSON decoding non-string",
-			expr: "body.pull_request.parseJSON().test == 'test'",
-			want: "unexpected type 'map' passed to parseJSON",
-		},
-		{
-			name: "parseYAML decoding non-string",
-			expr: "body.pull_request.parseYAML().key1 == 'value1'",
-			want: "unexpected type 'map' passed to parseYAML",
-		},
+		//{
+		//	name: "base64 decoding non-string",
+		//	expr: "body.pull_request.decodeb64()",
+		//	want: "unexpected type 'map' passed to decodeB64",
+		//},
+		//{
+		//	name: "parseJSON decoding non-string",
+		//	expr: "body.pull_request.parseJSON().test == 'test'",
+		//	want: "unexpected type 'map' passed to parseJSON",
+		//},
+		//{
+		//	name: "parseYAML decoding non-string",
+		//	expr: "body.pull_request.parseYAML().key1 == 'value1'",
+		//	want: "unexpected type 'map' passed to parseYAML",
+		//},
```

Comment:
@bigkevmcd I have commented this for now in order to inform that with latest CEL version once we started using `cel.Function` if there something not defined it will give below errors
```
    --- FAIL: TestExpressionEvaluation_Error/base64_decoding_non-string (0.00s)
        cel_test.go:794: evaluate() got expression "body.pull_request.decodeb64()" failed to evaluate: no such overload: decodeb64(map), wanted unexpected type 'map' passed to decodeB64
    --- FAIL: TestExpressionEvaluation_Error/parseJSON_decoding_non-string (0.00s)
        cel_test.go:794: evaluate() got expression "body.pull_request.parseJSON().test == 'test'" failed to evaluate: no such overload: parseJSON(map), wanted unexpected type 'map' passed to parseJSON
    --- FAIL: TestExpressionEvaluation_Error/parseYAML_decoding_non-string (0.01s)
        cel_test.go:794: evaluate() got expression "body.pull_request.parseYAML().key1 == 'value1'" failed to evaluate: no such overload: parseYAML(map), wanted unexpected type 'map' passed to parseYAML
    --- FAIL: TestExpressionEvaluation_Error/marshalJSON_marshalling_string (0.01s)
```

Saying no Overload
So planning to remove those but wanted to confirm with you before i do that